system/modules/task/actions/edit.php

```diff
@@ -206,12 +209,13 @@ function edit_POST($w) {
     }
     
     $task->fill($_POST['edit']);
+    $task->rate = abs($task->rate) == 0 ? NULL : $task->rate;
```

Comment:
Also I don't think there's a "reject" functionality, afaik you make the changes and push again to this pull request...